bin/pitivi-env

```diff
@@ -14,12 +14,13 @@ SCRIPT=${BASH_SOURCE[0]:-$0}
 SCRIPTDIR=$(dirname $(realpath $SCRIPT))
 export FLATPAK_ENVPATH=$(realpath $SCRIPTDIR/../../)
 export CURRENT_GST=$FLATPAK_ENVPATH
+PITIVI_INSTALL_PATH=$(realpath $SCRIPTDIR/../)
```

Comment:
PATH usually means a collections of paths to search, maybe use REPO_DIR? In which case also rename SCRIPTDIR -> SCRIPT_DIR.
I see `realpath` does it already, but please remove the "/" at the end.
Move this line after SCRIPTDIR, it should be easier to follow.